js-pkg/sdk/src/main.ts

```diff
@@ -227,6 +229,10 @@ export class DocumentManager {
     return result
   }
 
+  public async checkStore(): Promise<CheckStoreResult> {
+    return (await this.doFetch('check_store', 'GET')).json()
```

Comment:
Nit: I think it's a good practice to await the Promise returned from the `json()`, too. The reason is if someone wraps `checkStore()` in a try/catch, a json parsing error won't get caught unless it is awaited before returning.